### DISCUSSION TITLE
Very simple demo of Aperture

### DISCUSSION THREAD

**Discussion by sugarjig** [lightninglabs/aperture#118] [Show and tell]
Hi! I was curious to see Aperture in action, so I made a barebones demo of it being used by a trivial web service and a React web app that I modified from lnc-web's [connect-demo](https://github.com/lightninglabs/lnc-web/tree/main/demos/connect-demo) app. There is not much to the UI, and I made no attempt to handle errors, but you can use it to get an idea for how L402 works. You can find it here: https://github.com/sugarjig/l402-demo. Let me know what you think!

**Comment by sugarjig** [comment]
@hieblmi I saw a comment from you but don't see it anymore. I had to make the repo public. Please try again!

**Reply by hieblmi** [reply]
Deleted my comment after I saw that the repo link started working.
